docs/reference-guides/data/data-core.md

```diff
@@ -178,7 +178,7 @@ _Parameters_
 
 _Returns_
 
--   `undefined< EntityRecord > | undefined`: The entity record, merged with its edits.
```

Comment:
This `undefined< EntityRecord >` type exposes a bug in the `docgen` tool. The type in the source is `ET.Updatable<EntityRecord>`, and is imported with `import type * as ET from './types';`. Apparently the tool is unable to follow the wildcard references.

After I change the import to be non-wildcard (`import { Updatable, ... } from './types'`) then `docgen` can process it correctly.

Reply:
Fixing this in #61097.